docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -100,10 +100,40 @@ spec:
       resources:
         requests:
           storage: 1Gi
+---
+kind: PersistentVolume
```

Comment:
Another way is to use `StorageClass`, but i am not sure whether it will be too advanced for concepts tutorial.
- reduce number of replicas from 3 to 2 so that the YAML file will not be too long
- explicitly add the 2 PVs to ensure all pods are running

Reply:
Remove the persistent volumes.

Reply:
If we remove the PVs, a user who is new to statefulset may not know that there is a need to create the PVs too. The pods will be pending forever if the PVs are not created.

Reply:
As discussed by person, the concepts tutorials are not meant to have the YAML file cut and pasted. The statefulset should create the PVs automatically when dynamic provisioner is available.